src/mcts/node.h

```diff
@@ -306,7 +310,7 @@ class Node {
 #if defined(__i386__) || (defined(__arm__) && !defined(__aarch64__))
 static_assert(sizeof(Node) == 48, "Unexpected size of Node for 32bit compile");
```

Comment:
I'm guessing this needs updating to 52.

Reply:
I increased it to 52, but can't check if it's correct.

Reply:
Yeah, I'm confident its correct.  Adding a 32 bit field on 32 bit is much easier to guess what is going to happen than adding a 32 bit field on 64 bit.